doc-content/jbpm-docs/src/main/asciidoc/BPMN2/bpmn-intermediate-events-ref.adoc

```diff
@@ -1,7 +1,14 @@
 [id='bpmn-intermediate-events-ref_{context}']
 
 = Intermediate events
-Intermediate events drive the flow of a business process. Intermediate events are used to either catch or throw an event during the execution of the business process.  These events are placed between the start and end events and can also be used on the boundary of an activity, like a subprocess or a human task, as a catch event.  The boundary catch events can be configured as interrupting or non-interrupting.  An interrupting boundary catch event cancels the bound activity whereas a non-interrupting event does not.
+Intermediate events drive the flow of a business process. Intermediate events are used to either catch or throw an event during the execution of the business process. These events are placed between the start and end events and can also be used on the boundary of an activity, like a subprocess or a human task, as a catch event. In the BPMN modeler, you can set a data output in the *Data Output and Assignments* field for a boundary event, which is used in a further process to access the process instance details. For example, you can set the following data output variables for a boundary event:
```

Comment:
Trivial, but a line break so that "For example, ..." starts a new line would be helpful to set up the bullet list that follows. It's typically best practice for readability to set up bullet lists with very clear introductory sentences. If the intro sentence for the list is buried at the end of a paragraph, not as easy to see quickly what the list is of.

Reply:
Done